piton/tools/src/sims/sims,2.0

```diff
@@ -2900,6 +2901,9 @@ sub parse_args
   #
   push (@{$opt{sim_run_args}}, "-gui") if ($opt{gui}) ;
   push (@{$opt{vcs_build_args}}, "-debug_all") if ($opt{debug_all}) ;
+  push (@{$opt{vcs_build_args}}, "-debug_pp") if ($opt{debug_pp}) ;
+  push (@{$opt{vcs_build_args}}, "-CFLAGS \"-I/mnt/users/ssd1/homes/svijay97/BlackParrot/black-parrot-sdk/include -std=c++14\"") ;
```

Comment:
I'll change the path to a generic one

Reply:
Please do. It's also not being conditionally added?